pkg/TIPA.py

```
@@ -164,587 +165,498 @@ def __init__(self, img, mouse_pos, display_info_init, gui_queue):
+        if page1 is None or page1.status_code != 200:
+            if self.debug_mode >= 1:
+                logging.info("Unable to find tarkov-market page")
+            return None, None
 
-                    # Break the loop as we've found the item information
-                    if self.debug_mode >= 1:
-                        print("Found! Breaking")
+        # Scrape the gamepedia item webpage for more item details
+        try:
+            URL2 = f"https://escapefromtarkov.gamepedia.com/{true_name}"
+            page2 = requests.get(URL2)
```

Ruff (0.8.0)

Probable use of `requests` call without timeout

(S113)